### PR TITLE
Add Minecraft: New Nintendo 3DS Edition

### PR DESCRIPTION
**Pull request by JaroslavN**
Adds Minecraft: New Nintendo 3DS Edition.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/minecraft-new-nintendo-3ds-edition/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
+title = "Minecraft: New Nintendo 3DS Edition"
+description = "Minecraft is a game about placing blocks and going on adventures.\n\nOnly playable on the New Nintendo 3DS series.\n\nExplore randomly‐generated worlds and build amazing things from the simplest of homes to the grandest of castles. Play in Creative Mode with unlimited resources or mine deep into the world in survival mode, crafting weapons and armor to fend off the dangerous mobs."
```
Comment (Schplee) [inline games/minecraft-new-nintendo-3ds-edition/game.dat]: Remove the line breaks `\n`

File: games/minecraft-new-nintendo-3ds-edition/game.dat [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+region = "usa"
+release_date = "2017-09-13"
+
+[[testcases]]
+title = "00040000001B8700"
+compatibility = "5"
+date = "2017-09-17"
+version = "HEAD-255fd87"
+cpu = "Intel® Core™ i7-3635QM"
+gpu = "AMD Radeon™ HD 8770M Graphics"
```
Comment (Schplee) [inline games/minecraft-new-nintendo-3ds-edition/game.dat]: Remove `Graphics`

File: games/minecraft-new-nintendo-3ds-edition/game.dat [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+os = "Windows 10 Version 10.0.15063"
```
Comment (Schplee) [inline games/minecraft-new-nintendo-3ds-edition/game.dat]: Should be `Windows 10 15063`

**Comment by TheKoopaKingdom** [comment]
@JaroslavN please fix the issues mentioned above, or this PR will be closed.

**Pull request closed without merging**